assets/global.js

```diff
@@ -343,6 +343,8 @@ class MenuDrawer extends HTMLElement {
   }
 
   closeMenuDrawer(event, elementToFocus = false) {
+    if (event === undefined) return;
```

Comment:
This fixes the issue, but the root cause is in the function below. When a filter is selected, the focus shifts out of the drawer which causes `onFocusOut()` to trigger. By adding this condition to`closeMenuDrawer()`, `onFocusOut()` will now never actually close the dialog, since no event is passed as param.
``` js
onFocusOut(event) {
  setTimeout(() => {
    if (this.mainDetailsToggle.hasAttribute('open') && !this.mainDetailsToggle.contains(document.activeElement)) this.closeMenuDrawer();
  });
}
  ```